src/Makefile.am

```diff
@@ -97,6 +97,7 @@ fwts_SOURCES = main.c 				\
 	acpi/plddump/plddump.c			\
 	acpi/pmtt/pmtt.c 			\
 	acpi/rsdp/rsdp.c			\
+	sbbr/rsdp/rsdp.c			\
```

Comment:
is it difficult to make sbbr specific changes in acpi/rsdp/rsdp.c? If there a lot of changes, that's a valid argument to make when it is sent to maintainers.